client/src/components/pages/Results/index.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import axios from 'axios';
+import { notification } from 'antd';
+
+import Card from '../../common/Card';
+
+import './style.css';
+
+export default class ResultsPage extends React.Component {
+  state = {
+    therapist: [],
+    type: '',
+  };
+
+  async componentDidMount(types = 'PD') {
+    try {
+      const result = await axios.post('/api/v1/filter', {
+        data: { types },
+      });
+      const therapist = result.data.data;
+      this.setState({ therapist });
```

Comment:
using set state twice !! 
try to put all these thing in a single line 
```suggestion
      this.setState({ therapist , type: types });
```